scripts/tools-versions-darwin.yml

```diff
@@ -15,8 +15,8 @@ west:
 nanopb:
   version: 0.4.6
 zephyr-sdk:
-  version: 0.16.5
-  architectures: # https://github.com/zephyrproject-rtos/sdk-ng/releases/tag/v0.16.5
+  version: 0.16.5-1
```

Comment:
Why this bump ?

The only difference between 0.16.5 and 0.16.5-1 is aarch64, which were excluded in the 0.16.5 because of aws issues.

Reply:
I agree, not needed.

Reply:
@shanthanordic want to keep align with latest sdk-ng